azkaban-common/src/main/java/azkaban/jobExecutor/ProcessJob.java

```diff
@@ -259,7 +259,6 @@ public void run() throws Exception {
         }
         throw new RuntimeException(e);
       } finally {
-        this.process = null;
```

Comment:
@reallocf i don't think it's necessary. It would make getProgress() return 0 instead 1 as expected when process finishes.